src/types/api.ts

```diff
@@ -246,7 +246,7 @@ export type Consultation = EscolaLms.Consultations.Models.Consultation & {
   is_ended?: boolean;
   is_started?: boolean;
   in_coming?: boolean;
-  author: User & { categories: Category[] };
+  author: User & { categories: Category[]; bio: string | null };
```

Comment:
`bio` is additional fields 

additional fields can be anything so
```suggestion
  author: User & { categories: Category[] } & Record<string, any>
```
might be better 

or might be enough 

```suggestion
  author: User & { categories: Category[] } & Record<string, string | number | boolean>
```